src/java.base/share/classes/java/lang/Float.java

```diff
@@ -115,14 +115,30 @@ public final class Float extends Number
      */
     public static final float MIN_VALUE = 0x0.000002P-126f; // 1.4e-45f
 
+    /**
+     * The number of bits used to represent a {@code float} value.
+     *
+     * @since 1.5
+     */
+    public static final int SIZE = 32;
+
+    /**
+     * The number of bits in the significand of a {@code float} value.
+     * This is the parameter N in section {@jls 4.2.3} of
+     * <em>The Java Language Specification</em>.
```

Comment:
Please use <cite> rather than <em> tags for references to the JLS.

Reply:
Ready